config/helpers.php

```diff
@@ -449,12 +449,12 @@ function kirbytags(?string $text = null, array $data = []): string
  * given string. Shortcut for `$kirby->kirbytext()`
  *
  * @param string|null $text
- * @param array $data
+ * @param array $options
```

Comment:
Wondering if we need to consider this a breaking change with named parameters in PHP 8.0 and higher?

Reply:
I reverted those. They are not necessary in a minor release.